<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminpersonal</class>
 <widget class="QDialog" name="adminpersonal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <widget class="QTabWidget" name="tabwidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>48</y>
     <width>511</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_teacher">
    <attribute name="title">
     <string>教师管理</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0" colspan="5">
      <widget class="QScrollArea" name="scrollAreaTeacher">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>463</width>
          <height>204</height>
         </rect>
        </property>
        <widget class="QTableWidget" name="tabWidget_teacher">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>461</width>
           <height>211</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnDelTea">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>103</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnAddTea">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>103</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnReTea">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_stu">
    <attribute name="title">
     <string>学生管理</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0" colspan="5">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>463</width>
          <height>204</height>
         </rect>
        </property>
        <widget class="QTableWidget" name="tabWidget_stu">
         <property name="geometry">
          <rect>
           <x>-5</x>
           <y>1</y>
           <width>471</width>
           <height>201</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnAddStu">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>103</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>103</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnRefreshStu">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnDelStu">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_exam">
    <attribute name="title">
     <string>考卷管理</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_exam">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>11</y>
       <width>501</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>增加考试</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="addpapername"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>20</y>
     <width>301</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>当前科目</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="currentPaper">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnsubmit">
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
